### PR TITLE
Functional tests - Add new test FO > User account > 'Check gdpr data in pdf' 

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.0.x
| Description?      | Add new test FO> User account > 'Check gdpr data in pdf' 
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| Related PRs       | no
| How to test?      | no need
| Possible impacts? | no

### CODE REVIEW

File: tests/UI/utils/files.js [hunk 1]
```diff
@@ -66,14 +66,14 @@ module.exports = {
     }
 
     const pageTexts = await Promise.all(pageTextPromises);
-
+    console.log(pageTexts.join(' '));
```
Comment (Progi1984) [inline tests/UI/utils/files.js]:
```suggestion
```

**Review by Progi1984 — COMMENTED** [review]
A small `console.log` :wink: and it's approved.

Just a :clap: before it's a beautiful job.

File: tests/UI/utils/files.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/utils/files.js]: Done :)

**Review by Progi1984 — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
2 approvals but CI seems to spin forever... 🤔

**Comment by matthieu-rolland** [comment]
thank you @nesrineabdmouleh !

**Pull request merged**